x-pack/plugins/synthetics/public/apps/synthetics/components/settings/data_retention.tsx

```diff
@@ -76,6 +91,65 @@ export const DataRetentionTab = () => {
   );
 };
 
+const Unprivileged = () => {
+  const {
+    services: { docLinks },
+  } = useKibana();
+  return (
+    <EuiEmptyPrompt
+      data-test-subj="syntheticsUnprivileged"
+      color="plain"
+      icon={<EuiIcon type="logoObservability" size="xl" />}
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.synthetics.noFindingsStates.unprivileged.unprivilegedTitle"
+            defaultMessage="Privileges required"
+          />
+        </h2>
+      }
+      body={
+        <p>
+          <FormattedMessage
+            id="xpack.synthetics.params.unprivileged.unprivilegedDescription"
+            defaultMessage="To view Synthetics monitors data usage and retention, you must update privileges. For more information, contact your Kibana administrator. {docsLink}"
+            values={{
+              docsLink: (
+                <EuiLink
+                  data-test-subj="syntheticsUnprivilegedLearnMoreLink"
+                  href={docLinks?.links.synthetics.featureRoles}
+                  target="_blank"
+                >
+                  {i18n.translate('xpack.synthetics.monitorManagement.projectDelete.docsLink', {
+                    defaultMessage: 'Learn more',
+                  })}
+                </EuiLink>
+              ),
+            }}
+          />
+        </p>
+      }
+      footer={
+        <EuiMarkdownFormat
+          css={css`
+            text-align: initial;
+          `}
+          children={`\n- ${INDEX_PRIVILEGES} \n- ${CLUSTER_PRIVILEGES}`}
+        />
+      }
+    />
+  );
+};
+
+const INDEX_PRIVILEGES = i18n.translate('xpack.synthetics.dataRetention.unprivileged.index', {
+  defaultMessage:
+    'Required Elasticsearch index privilege `read`, `monitor` for the following indices: `synthetics-*`',
+});
+
+const CLUSTER_PRIVILEGES = i18n.translate('xpack.synthetics.dataRetention.unprivileged.cluster', {
+  defaultMessage: 'Required Elasticsearch cluster privilege `read_ilm`, `monitor`.',
```

Comment:
The required cluster privilege is actually `manage_ilm`. With read ilm, they'll be able to see the ilm policies but when they click on them they'll be taking to an empty page.

Reply:
My preference would be to only show this page when they have `manage_ilm`. If that's challenging, then we should not hyperlink the ilm policies when user only has `read_ilm` (as the hyperlinks won't work). Whatever is easiest to do.